daemon/lib/source/DobbyManager.cpp

```diff
@@ -2373,7 +2374,6 @@ void DobbyManager::onChildExit()
 
             AI_LOG_INFO("runc for container '%s' has quit (pid:%d status:0x%04x)",
                         id.c_str(), containerPid, status);
-
```

Comment:
If you didn't change anything around this place don't make style adjustments as it will result in possible merge conflicts.